### PR TITLE
include a statement about technologies used in intro

### PR DESCRIPTION
**Pull request by fgeorgatos**

### CODE REVIEW

File: docs/introduction/index.rst [hunk 1]
```diff
@@ -115,6 +120,8 @@ accidental data duplication as much as possible.
 History
 -------
 
+- 2018.08.01. **Renku** Component Architecture is solidifying now to a more long-term design
```
Comment (rokroskar) [inline docs/introduction/index.rst]: I wand to add a line here when we actually do a release, so I think we can remove for now

File: docs/introduction/index.rst [hunk 2]
```diff
@@ -19,7 +19,12 @@ Features
 --------
 
 Renku consists of a collection of services, including a web-based user interface
-and a command-line client.
+and a command-line client, exploiting in a coherent setup the joint features of:
+* GitLab - version control
+* JupyterHub - Interactive Notebooks
+* Kubernetes - Container Orchestration
+* Keycloak - Identity and Access Management
+* Python - Scripting, glue logic and much more...
```
Comment (rokroskar) [inline docs/introduction/index.rst]: remove "and much more"

File: docs/introduction/index.rst [hunk 3]
```diff
@@ -19,7 +19,12 @@ Features
 --------
 
 Renku consists of a collection of services, including a web-based user interface
-and a command-line client.
+and a command-line client, exploiting in a coherent setup the joint features of:
+* GitLab - version control
```
Comment (rokroskar) [inline docs/introduction/index.rst]: gitlab is actually for repository management -- version control is handled by git. Could we just say `GitLab - repository management`?

Comment (rokroskar) [inline docs/introduction/index.rst]: need a blank line to get the list

File: docs/introduction/index.rst [hunk 4]
```diff
@@ -19,7 +19,13 @@ Features
...
 
 Renku consists of a collection of services, including a web-based user interface
-and a command-line client.
+and a command-line client, exploiting in a coherent setup the joint features of:
+
+* GitLab -  repository management
+* JupyterHub - Interactive Notebooks
+* Kubernetes - Container Orchestration
+* Keycloak - Identity and Access Management
+* Python - Scripting & glue logic
```
Comment (rokroskar) [inline docs/introduction/index.rst]: hmm I don't know about including python here -- it's not really a technology. It's clearly different than the other things on the list so I would remove it. Maybe also add Common Workflow Language and hyperlinks to the web pages for these components?

**Comment by fgeorgatos** [comment]
@rok: thanks for the markup improvement - merged! I feel for you, with the markup format fighting :)

**Pull request merged**